code/modules/mob/living/living.dm

```diff
@@ -481,7 +481,10 @@
 /mob/proc/get_contents()
 
 
-//Gets ID card from a mob. If hand_firsts is TRUE hands are checked first, otherwise other slots are prioritized (for subtypes at least).
+/**
+ * Gets ID card from a mob.
+ * If hand_firsts is TRUE hands are checked first, otherwise other slots are prioritized (for subtypes at least).
+ */
```

Comment:
you should add a line for specifically what is returned here, and heck even throw in a 
```dm
 * Arguments:
 * * hand_first - boolean that checks the hands of the mob first if TRUE
```